Project-Hotpot/Project-Hotpot/Algorithm/SongManager.swift

```diff
@@ -27,6 +51,7 @@ class SongManager : NSObject {
                 return completion(.failure(error))
             }
         }
+        self.historySet.insert(spotifySong.uri)
```

Comment:
Is there an upper bound to how this can grow, for example if i keep switching all the songs? I thought somewhere in design you had mentioned you only want to consider the last 100 or so songs?